test-it/ViaTests/ArrayMaxMin.via

```diff
@@ -270,11 +270,11 @@ define (_%54estVI_ArrayMaxMin%2Egvi dv(.VirtualInstrument (
         Copy(local170 local135 )
         Copy(local135 di_2407 )
         ArrayMaxMin(g41 local137 local138 local139 local140 )
-Println(g41)
-Println(local137)
-Println(local138)
-Println(local139)
-Println(local140)
+        Println(g41)
+        Println(local137)
+        Println(local138)
+        Println(local139)
+        Println(local140)
```

Comment:
We should add more tests for all comparison operators and other primitives that use comparison operators. We should come up with contrived data with different sized integer data types that will fail if they have bits representing nan/infinity/-infiniity/etc when they were treated as float/double incorrectly.

Better to write such tests in G and check in VIA here.

Reply:
For the same reason as my previous comment, I would prefer to do this in a different commit.